Update/omot_016.txt

```diff
@@ -2156,7 +2156,7 @@ void main()
 //r野菜がいやに少なく見えるのは、きっと売り物にならなくなったためだろう。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "野菜がいやに少なく見えるのは、きっと売り物にならなくなったためだろう。",
-			NULL, "It looked like far fewer vegetables than usual, so they were probably not open for business.", Line_Normal);
+			NULL, "It seemed the reason they had far fewer vegetables than usual was because the rest had been damaged beyond salvaging.", Line_Normal);
```

Comment:
```suggestion
			NULL, "It seemed the reason they had far fewer vegetables than usual was because the rest had been damaged beyond salvage.", Line_Normal);
```